src/components/editor/property/LanguageButton.jsx

```diff
@@ -48,7 +48,8 @@ const LanguageButton = (props) => {
       <Button
         id="language"
         bsSize="small"
-        onClick = { () => setShow(true) }>
+        onClick = { () => setShow(true) }
+        className="btn btn-literal btn-default">
```

Comment:
Button should not have a `btn-default` className.  Instead use the `bsStyle="default"`

Reply:
Fixed @jcoyne